ingestions/ingest_csvs.py

```diff
@@ -213,31 +378,50 @@ def main(argv=None):
                         help="Ingest state, either run, stage or mock (default: run)")
     parser.add_argument("-y", "--default_review", dest="default_review", default=False,
                         action="store_true", help="Set the ingest reviews to default to yes, skipping the review.")
+    parser.add_argument("--debug", dest="debug", default=False,
+                        action="store_true",
+                        help="A dry run that prints the final json "
+                            "ingest request instead of actually submitting "
+                            " to the api for debugging purposes.")
     parser.add_argument("-am", "--use_am_dates", dest="am_dates", default=False,
                         action='store_true', help="Use Asset Management to set start and end dates")
     parser.add_argument("-bd", "--begin_date", dest="begin_date", type=str,
                         help="Date and/or time string to set the start time of the data")
     parser.add_argument("-ed", "--end_date", dest="end_date", type=str,
                         help="Date and/or time string to set the end time of the data")
+    parser.add_argument("-st", "--server-target", dest="server_target", type=str,
+                        choices=('prod', 'dev01', 'dev02'), default='prod',
+                        help="Select the server to ingest data into (default: prod)")
+    parser.add_argument("-u", "--user", dest="netrcaccount", type=str,
+                        default=PROD_DOMAIN,
+                        help= """The user netrnc "machine" name to use 
+        for authentication.  Either the full machine name or the 
```

Comment:
Needs consistent indentation

Reply:
That is why I used the triple strings so I didn't have to stack all of the long string on multiple lines packed to the right side.  Regardless, fixed now.